src/amber/router/params.cr

```diff
@@ -1,129 +1,4 @@
-require "json"
-
 module Amber::Router
-  # The Params module will parse parameters from a URL, a form post or a JSON
```

Comment:
Why was everything but 4 lines deleted from this file and copied to params_parser?

Reply:
The rename of the file is necessary here since it adds context on what the file does.